SQL/database_changelog.txt

```diff
@@ -1,15 +1,22 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 5.11; The query to update the schema revision table is:
+The latest database version is 5.12; The query to update the schema revision table is:
```

Comment:
Another place this needs to be changed in https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/subsystems.dm#L23